runtime/doc/indent.txt

```diff
@@ -570,9 +570,15 @@ The examples below assume a 'shiftwidth' of 4.
 	      with "#" does not work.
 
 
+	PN    When N is non-zero recognize C pragmas, does not concern other
```

Comment:
I think it is unclear that it treats pragmas as normal code.

Reply:
Yeah, I improved it now.